design-editor/src/system/stage-manager.js

```diff
@@ -260,11 +262,18 @@ class StageManager {
 
             if (window.atom !== undefined) {
```

Comment:
We don't support atom anymore. You can refactor this if .. else statement and stay DRY rule